libs/facilities/frontend/view/src/lib/components/issues/issues.page.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ActivatedRoute } from '@angular/router';
+
+import { XdIssuesPage } from './issues.page';
+
+describe('IssuesComponent', () => {
+	let component: XdIssuesPage;
+	let fixture: ComponentFixture<XdIssuesPage>;
+
+	beforeEach(async () => {
+		await TestBed.configureTestingModule({
+			imports: [XdIssuesPage],
+			providers: [
+				{
+					provide: ActivatedRoute,
+					useValue: {},
+				},
+			],
+		}).compileComponents();
+
+		fixture = TestBed.createComponent(XdIssuesPage);
+		component = fixture.componentInstance;
+		fixture.detectChanges();
+	});
+
+	it('should create', () => {
+		expect(component).toBeTruthy();
+	});
+});
```

Comment:
Where are tests?

Reply:
The issue page right now is just the browse page and will be changed to look different in the issue
https://github.com/orgs/amosproj/projects/43/views/2?pane=issue&itemId=62977084

In general we often don't know if the site actually should stay like it is right now. So adding tests early does not really make sense in my opinion